src/components/FoodBox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class FoodBox extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            count: 1
+        }
+    }
+    handleQuantity = (event) =>{
+        this.setState({
+            count: event.currentTarget.value
+        })
+    }
+    render(){
+        return (
+            <div className="box">
+                <article className="media">
+                    <div className="media-left">
+                    <figure className="image is-64x64">
+                        <img src={this.props.food.image} alt='img'/>
+                    </figure>
+                    </div>
+                    <div className="media-content">
+                    <div className="content">
+                        <p>
+                        <strong>{this.props.food.name}</strong> <br />
+                        <small>{this.props.food.calories}</small>
+                        </p>
+                    </div>
+                    </div>
+                    <div className="media-right">
+                    <div className="field has-addons">
+                        <div className="control">
+                        <input onChange = {this.handleQuantity} className="input" type="number" value={this.state.count} />
```

Comment:
this is a perfect example of a dynamic react input that changes via the state, and the you call the onAddToday function with the state. So perfectly done :)